ckine/figures/figure6.py

```diff
@@ -278,6 +281,7 @@ def Spec_Aff(ax, npoints, unkVecAff):
     ax.set_xlabel("Relative IL-2Rβ/γc $K_a$")
     ax.set_ylabel("Specificity")
     ax.set_xlim((10e-3, 10e0))
+    ax.set_yscale("log")
```

Comment:
Does this need to be log-scaled?

Reply:
Yea unfortunately it does - now that the NK cell predictions happen to be quite a bit larger than that of Tregs and Thelpers, without the log scale the trend is very difficult to see as NK/Treg is very small.